src/test/java/org/opentripplanner/middleware/testUtils/CommonTestUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.opentripplanner.middleware.testUtils;
+
+import org.opentripplanner.middleware.utils.FileUtils;
+
+import java.io.IOException;
+
+public class CommonTestUtils {
+    /**
+     * Whether the end-to-end environment variable is enabled.
+     */
+    public static final boolean isEndToEnd = getBooleanEnvVar("RUN_E2E");
```

Comment:
I think maybe this should be all caps/underscore since it's static/final.

Reply:
Changed in 695f54c4ffb08e23144a496d611b999c93c86764.